Version Control/Git Commands.md

```diff
@@ -0,0 +1,88 @@
+# Git Commands Reference Guide
+
+When working with Git, it's inevitable you'll have to work with the command line. This guide will help you with the most common commands you'll need to know. For more information on a specific command, you can use the `git help` command followed by the command you want to learn more about. For example, `git help commit`. There is also the [Git Guide](https://public-health-scotland.github.io/git-guide), which is a more in-depth guide to using Git in PHS, including a more complete list of commands.
+
+Note that Git commands work on the system-level so need to be entered on a terminal, not the language console. For example, in RStudio, the terminal is likely open as a tab beside the console (`Alt` + `Shift` + `M`), or a new one can be opened in Tools > Terminal > New Terminal (`Alt` + `Shift` + `R`). Console's usually have `>` as the line starter, where the terminal will use a `$`. This will look like:
```

Comment:
Thanks for adding this! Just one rogue apostrophe in 'Console's' but otherwise I think this is clear for users.